modules/UpgradeWizard/silentUpgrade_step1.php

```diff
@@ -375,7 +386,7 @@ function threeWayMerge()
 //Clean_string cleans out any file  passed in as a parameter
 $_SERVER['PHP_SELF'] = 'silentUpgrade.php';
 
-$usage_regular =<<<eoq2
+$usage_regular = <<<eoq2
 Usage: php.exe -f silentUpgrade.php [upgradeZipFile] [logFile] [pathToSugarInstance] [admin-user]
```

Comment:
@Dillon-Brown  I propose to replace 'Sugar' with 'SuiteCRM':

> Usage: php.exe -f silentUpgrade.php [upgradeZipFile] [logFile] [pathToSuiteInstance] [admin-user]
> On Command Prompt Change directory to where silentUpgrade.php resides. Then type path to
> php.exe followed by -f silentUpgrade.php and the arguments.  
> 
> Example:  
>     [path-to-PHP/]php.exe -f silentUpgrade.php [path-to-upgrade-package/]SuiteCRM-Upgrade-7.0.x-to-7.11.7. .zip [path-to-log-file/]silentupgrade.log  [path-to-suite-instance/] admin
> 
> Arguments:
>     upgradeZipFile                 : Upgrade package file.
>     logFile                              : Silent Upgarde log file.
>     pathToSuiteInstance         : SuiteCRM instance being upgraded.
>     admin-user                      : admin user performing the upgrade
>

Reply:
Done, thanks!